notification/mail.py

```diff
@@ -111,12 +111,16 @@
       - The character set of email being sent
     default: 'us-ascii'
     required: false
+  subtype:
```

Comment:
Well, it is not really a mimetype you have to provide, maybe mimesubtype would be preferable ?
I prefer subtype over mimetype (as it is more correct), cfr. https://docs.python.org/2/library/mimetypes.html

Whatever the decision is, I support this change. +1